### ISSUE TITLE
SAR not working with Reaper 

### ISSUE THREAD

**Issue by Darko0089** [eiz/SynchronousAudioRouter#48]

**Comment by pannal** [comment]
I can confirm that. I've recently (painfully) rolled back the latest Windows Update because SAR+REAPER didn't work anymore.

**Comment by rainbowjaw** [comment]
+1 not working. I've been screwing around with a bunch of audio configs and having a hard time... maybe this update is a the root of a lot of issues.

**Comment by AKPWebDesign** [comment]
I'm on Windows 10 Insider Fast Ring, so it updates a lot. After every update, it seems that Windows screws something up with the SAR installation, and it needs to be reinstalled. After a reinstall though, everything works fine.

**Comment by pannal** [comment]
@AKPWebDesign how's the reinstall process? Is it enough to just install it over the current installation, or does it have to be uninstalled -> reboot -> install -> reboot?

**Comment by AKPWebDesign** [comment]
@pannal Usually just an in-place reinstall works for me (uninstall and reinstall with no reboot). I tried the repair option once and it didn't work, but that might have been a failure somewhere else. Either way, the reboot isn't necessary in my experience, because nothing on the system has been able to actually initialize and use SAR properly thanks to whatever Windows is doing to screw up the install.

**Comment by pannal** [comment]
@AKPWebDesign OK, interesting. Before I rolled the upgrade back, I had the issue that REAPER was unable to open the ASIO interface - did you experience the same? I feared that ASIO4ALL was also a culprit, besides SAR.

**Comment by AKPWebDesign** [comment]
@pannal Yeah, that issue (unable to open the ASIO interface) was the reason I tried to reinstall SAR in the first place. I use a Focusrite Scarlett, and REAPER was able to access it directly just fine, but was unable to use SAR until the reinstall.

**Comment by pannal** [comment]
@AKPWebDesign thanks, I can confirm that 1803 works with SAR, ASIO4ALL and REAPER. An uninstall+reinstall of SAR fixes all issues after upgrading.

**Comment by lamskoy** [comment]
Doesn't work with 4i4 3rd gen too and Reaper
